Identifying the gene expression patterns associated with the response to the Prevnar-13 vaccine through transcriptomics					

This analysis involes trascriptomoic study of Prevnar-13 which is a vaccine against pneumococcal disease.
It covers 13 serotypes responsible for invasive pneumococcal disease
children aged 6 weeks to 5 years  and adults ≥50 years are admistered with the vaccine, especially those at risk for pneumococcal disease.


The immune response involves:
1. chemically linked to a carrier protein MHC class II molecules.
2. CD4+ T-helper cells co-stimulatory signals to B cells
3. B cells undergo:
      Class switching
      Affinity maturation
      Plasma cells and memory B cells.
   
Microarray:
The data is extracted in this analysis is from Microarray.
Microarray has a solid surface with fixed DNA probes, purified mRNA is converted to labelled cDNA and allowed to bind against the fixed probes.
Post this step singles are captured and gene expression is measured.
The liink of data set:  https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE298577

<img width="439" height="526" alt="Screenshot 2025-09-18 at 7 16 54 PM" src="https://github.com/user-attachments/assets/63694fa8-44a5-4f3e-b902-9e7078dc4fdc" />

The Analysis involves 3 Major segments.
1.Dimensionality Reduction & Clustering
            PCA
            UMAP
            t-SNE
2. Differential Expression & Enrichment
            Volcano Plot
            Heat Map
            CERNO Results
3.Pathway-Level Insights
            GSEA (Genome-Wide Association)
            Hallmarks of GSEA
            Tree Plot


In summary
Transcriptomic Analysis of Vaccine Response
Processed expression data and performed quality checks.
Conducted PCA and clustering to assess sample grouping.
Identified differentially expressed genes.
Performed pathway and module enrichment analysis.
Conducted GSEA using immunology (C7), vaccine-related (C7-VAX), and Hallmark gene sets.
Visualized findings with barplots, network plots, treeplots, and dotplots.
Conclusion: The study provides insights into vaccine-induced transcriptional changes by integrating differential expression and enrichment analyses.
